WcaOnRails/app/models/registration.rb

```diff
@@ -115,7 +115,8 @@ def paid_entry_fees
   end
 
   def last_payment_date
-    registration_payments.order(created_at: :desc).first&.created_at
+    sorted_payments = registration_payments.sort_by { |p| p.created_at }
+    sorted_payments.first&.created_at
```

Comment:
Since we're doing plain ruby here, can't we use `.max` instead of going to the effort of sorting? For example:

```ruby
registration_payments.map(&:created_at).max
```